region/crateria/central.json

```diff
@@ -5827,6 +6047,22 @@
                   "name": "Base",
                   "notable": false,
                   "requires": [ "Morph" ]
+                },
+                {
+                  "name": "Remote Acquire Item",
```

Comment:
3->1 using `C` and `h_canArtificialMorphDiagonalBombJump`

Reply:
That is listed as 3->4->1, reusing C.  Should be fine but might need to be cleaned up later.